Tools/cmake/FindDocGenerator.cmake.in

```diff
@@ -0,0 +1,17 @@
+find_package(JsonGenerator REQUIRED)
+
+function(DocGenerator)
+    set(oneValueArgs INPUT)
+
+    cmake_parse_arguments(Argument "${oneValueArgs}" ${ARGN} )
+
+    if(Argument_INPUT)
+        set(PLUGIN_NAME ${ARGN})
+    endif()
+
+    set(GENERATOR_SEARCH_PATH ${CMAKE_SYSROOT}${CMAKE_INSTALL_PREFIX}/include/${NAMESPACE})
+    set(CPPIFDIR_PATH ${CMAKE_SYSROOT}${CMAKE_INSTALL_PREFIX}/include/WPEFramework/interfaces/)
```

Comment:
Where is this method supposed to be invoked from? Plugin's makefile? Then this path won't lead to interface directory.